src/Http/Role/AddLabelToRoleRequestHandler.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Role;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\Request\RouteParameters;
+use CultuurNet\UDB3\Label\ReadModels\JSON\Repository\ReadRepositoryInterface;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use CultuurNet\UDB3\Role\Commands\AddLabel;
+use Fig\Http\Message\StatusCodeInterface;
+use InvalidArgumentException;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Slim\Psr7\Response;
+
+class AddLabelToRoleRequestHandler implements RequestHandlerInterface
+{
+    private CommandBus $commandBus;
+
+    private ReadRepositoryInterface $labelRepository;
+
+    public function __construct(CommandBus $commandBus, ReadRepositoryInterface $labelRepository)
+    {
+        $this->commandBus = $commandBus;
+        $this->labelRepository = $labelRepository;
+    }
+
+    public function handle(ServerRequestInterface $request): ResponseInterface
+    {
+        $routeParameters = new RouteParameters($request);
+        $roleId = $routeParameters->getRoleId();
+
+        try {
+            $roleId = new UUID($roleId);
+        } catch (InvalidArgumentException $e) {
+            throw ApiProblem::roleNotFound($roleId);
+        }
+
+        $labelIdentifier = $routeParameters->get('labelIdentifier');
+        try {
+            $labelId = new UUID($labelIdentifier);
+        } catch (InvalidArgumentException $exception) {
+            $entity = $this->labelRepository->getByName($labelIdentifier);
+
+            if ($entity === null) {
+                throw ApiProblem::blank('There is no label with identifier: ' . $labelIdentifier, 404);
+            }
+
+            $labelId = new UUID($entity->getUuid()->toString());
+        }
+
+        $this->commandBus->dispatch(new AddLabel($roleId, $labelId));
+
+        return new Response(StatusCodeInterface::STATUS_NO_CONTENT);
```

Comment:
We have a dedicated class `NoContentResponse` for this

Reply:
Haha oops sorry for the double comments @steftrenson , looks like Luc and I reviewed at exactly the same time :)